src/ontology/mondo-edit.obo

```diff
@@ -430932,13 +430935,166 @@ is_a: MONDO:0035561 {source="https://orcid.org/0000-0001-5208-3432", source="htt
 relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/9449 ! PRNP
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 
+[Term]
+id: MONDO:0035635
+name: short stature-skeletal dysplasia-retinal degeneration-intellectual disability-sensorineural hearing loss syndrome
+subset: ordo_disorder
+synonym: "Liberfarb syndrome" EXACT [Orphanet:589442]
+xref: Orphanet:589442 {source="MONDO:equivalentTo"}
+is_a: MONDO:0019755 {source="https://orcid.org/0000-0001-5208-3432"} ! developmental defect during embryogenesis
```

Comment:
this was a malformation syndrome in Orphanet and we obsoleted that and have a 'consider' annotation pointing to developmental defect during embryogenesis. @sabrinatoro do you think this is the best parent? Or maybe it should be syndromic disorder instead?